shepherd_bms/lib/LTC/Linduino.h

```diff
@@ -0,0 +1,128 @@
+//! @todo Review this file.
```

Comment:
Is this purely here for review and then later deletion?

Reply:
Oop nvm. Just noticed its in lib. I am open to keeping these if we think it's easier